client/src/app/site/modules/wait-for-action-dialog/components/wait-for-action-dialog/wait-for-action-dialog.component.html

```diff
@@ -1,24 +1,7 @@
 <div>
-    <h2>{{ currentTitle | translate }}</h2>
-    <p>{{ 'Worker identification' | translate }}: {{ currentWorkerId }}</p>
-    <p>{{ 'Action type' | translate }}: {{ currentWorkerName }}</p>
-    <p *ngIf="isPastInactivityThreshold">{{ 'Last confirmed activity on action' | translate }}: {{ lastActivity }}</p>
-    <p>{{ 'If you choose to stop waiting for this action, you will not receive results from the server.' | translate }}</p>
-    <p><i class="warn">{{ 'Warning: The process may still continue to run on the server, therefore actions that include creating, updating or deleting data will still do so eventually.' | translate }}</i><i *ngIf="isPastInactivityThreshold" class="warn">{{ 'This is, however, unlikely at this point.' | translate }}</i></p>
-    <div class="text-center">
-        <div class="wait-for-action-buttons">
-            <button mat-button (click)="wait()">
-                {{ 'Continue waiting' | translate }}
-            </button>
-            <button mat-button (click)="stop()">
-                {{ 'Stop waiting' | translate }}
-            </button>
-            <button mat-button [disabled]="!multiButtonNumber" (click)="wait(true)">
-                {{ multiWaitButtonLabel | translate }} {{multiButtonNumber}}
-            </button>
-            <button mat-button [disabled]="!multiButtonNumber" (click)="stop(true)">
-                {{ multiStopButtonLabel | translate }} {{multiButtonNumber}}
-            </button>
-        </div>
-    </div>
+    <h2>{{ 'Please wait' | translate }}</h2>
+    <p></p>
+    <p>{{ 'The server process is still running.' | translate }}</p>
```

Comment:
@emanuelschuetze @rrenkert  given that this dialog may be called when the action has been found to have stopped doing anything (i.e. the case where `isPastInactivityThreshold === true`) I find this message (and maybe also the inability to stop waiting) HIGHLY inappropriate